src/actions/passphrase.js

```diff
@@ -163,15 +168,33 @@ export const updatePassphrase2FASecond = (
         ) {
           dispatch({
             type: UPDATE_PASSPHRASE_2FA_2_FAILURE,
-            errors: { wrongTwoFactor: 'ProfileView.password.wrong_two_fa_code' }
+            errors: { wrongTwoFactor: 'PassphraseView.wrong_two_fa_code' }
           })
         } else {
           dispatch({
             type: UPDATE_PASSPHRASE_2FA_2_FAILURE,
-            errors: { global: 'ProfileView.password.server_error' }
+            errors: { global: 'PassphraseView.server_error' }
           })
         }
         throw error
       })
   }
 }
+
+export const updateHint = hint => {
+  return async dispatch => {
+    dispatch({ type: UPDATE_HINT })
+
+    try {
+      await cozyFetch('PUT', '/settings/hint', { hint })
```

Comment:
is it too complicated to start to use cozy-client here?

Reply:
I already consumed a lot of time on this PR, I'd rather not touch this part for now.